test/functional/api/v2/override_values_controller_test.rb

```diff
@@ -68,17 +68,14 @@ class Api::V2::OverrideValuesControllerTest < ActionController::TestCase
     assert_response :success
   end
 
-  [{ :value => 'xyz=10'}, { :match => 'os=string'}].each do |override_value|
```

Comment:
@amirfefer Not sure why did you remove the 2nd override test? This is generating two tests.